survey-sdk/src/main/java/co/nimblehq/surveysdk/entity/SurveyDetailEntity.kt

```diff
@@ -0,0 +1,59 @@
+package co.nimblehq.surveysdk.entity
+
+import com.squareup.moshi.Json
+
+data class SurveyDetailEntity(
+    val data: SurveyItem,
+    val included: List<SurveyItem>?
+) {
+
+    data class SurveyItem(
+        var id: String,
+        var type: String?,
+        var attributes: Attributes?,
+        var relationships: Relationships?
+    ) {
```

Comment:
Our api is built on [json-api](https://jsonapi.org/), I think we should try to define and parse our model/entity by [moshi-jsonapi](https://jsonapi.org/implementations/#client-libraries-android)

Reply:
@luongvo sorry, I did not get your point. 
Currently, I'm using Moshi to parse this model/entity, you want to put the tag @Json at every field in model/entity ❓

Reply:
@doannimble ICYDK, please double check the `json-api` link shared from Lucas to get familiar with the JSON:API format first. It's not just a simple JSON string as you are working on before. I think we could open a discussion to clarify this topic if you are still confused.

Reply:
@hoangnguyen92dn let me explore the `json-api` first, I'm wondering what it is 👀

Reply:
@luongvo @hoangnguyen92dn I updated the entities with `json-api` already, please help to check. Thank you for your support ❤️